src/pages/post-feed/CreatPost.tsx

```diff
@@ -0,0 +1,56 @@
+/* eslint-disable no-shadow */
+import Image from 'next/image';
+
+export default function CreatPost() {
+  interface User {
+    id: number;
+    userPhoto: string;
+  }
+
+  const user: User[] = [
```

Comment:
❌ 
this variable is called `user` but it has an array, which is confusing, the name should change to be something else like `users` or `userList`